.github/workflows/cibuildwheel.yml

```diff
@@ -91,11 +91,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CIBW_BEFORE_BUILD: >-
-        pip install numpy>=1.25 &&
         rm -rf {package}/build
       CIBW_BEFORE_BUILD_WINDOWS: >-
-        pip install delvewheel numpy>=1.25 &&
```

Comment:
I believe this was added when we were having problems with `oldest-supported-numpy` (and decided to fix them by switching to the np 1.25+ backwards compatible builds)

I think that it is likely we haven't needed it on this line since we switched to meson/pyproject.toml, but it did get changed in #26443.

Reply:
but if we are using build isolation the wheel build won't see this numpy instal....

Reply:
I thought we have build isolation by default in cibuildwheel, but I am not sure what the default does exactly.